pkg/proto/configuration/bb_worker/bb_worker.proto

```diff
@@ -13,6 +13,15 @@ option go_package = "github.com/buildbarn/bb-remote-execution/pkg/proto/configur
 
 message ApplicationConfiguration {
   // Configuration for blob storage.
+  //
+  // Note: some configurations have implmentation nounces that lead to a differences in runtime behavior
+  // between bb-storage acting as a bb-frontend and bb-worker, even while blobstore configuration is shared.
+  ///
+  // For example: buildbarn.configuration.grpc.ClientConfiguration.forward_metadata = ["authorization"]
+  // will work on the bb-storage frontend and forward client provided "authorization" header.
+  // In on the worker, the grpc call is terminated in the scheduler and when worker synchronizes with scheduler
```

Comment:
I'm not 100% sure does bb-frontend passes the context to bb-scheduler or it's terminated in bb-frontend.